clang/test/dpct/user_defined_rule.cu

```diff
@@ -158,4 +158,14 @@ __device__ void foo5(){ OldType<int> *cu_st;}
 __global__ void foo6(){
 //CHECK: std::ceil(2.5);
   std::ceil(2.5);
+}
+
+int my_max(int a, int b) {
+  return 0;
```

Comment:
Let's add readme in extensions/opt_rules to add guide user to add post processing rules and maybe other user defined rules